release/cloudbuild-release.yaml

```diff
@@ -4,10 +4,11 @@
 # See: https://cloud.google.com/cloud-build/docs/build-debug-locally
 # Then run:
 # cloud-build-local --config=cloudbuild-release.yaml --dryrun=false \
-#   --substitutions TAG_NAME=[TAG] .
+#   --substitutions TAG_NAME=[TAG],_INTERNAL_REPO_URL=[URL] .
 #
 # To manually trigger a build on GCB, run:
-# gcloud builds submit --config cloudbuild-proxy.yaml --substitutions TAG_NAME=[TAG] .
+# gcloud builds submit --config cloudbuild-proxy.yaml --substitutions \
```

Comment:
We should update this comment (as well as the rest of the cloudbuild-* files) to be "--config release/cloudbuild-proxy.yaml" right? I think we are assuming that this command is being run from the repo root.